getting_started_tutorials/cuml_sklearn_colab_demo.ipynb

```diff
@@ -0,0 +1,4310 @@
+{
```

Comment:
<blockquote>UMAP is a popular dimensionality reduction technique that is used to help visualize cluster very wide datasets</blockquote><p> </p><p>Suggestion:</p><p> </p><blockquote>UMAP is a popular dimensionality reduction technique that is used for both data visualization and as preprocessing for downstream modeling due to its ability to balance preserving both local and global structure of high-dimensional data. To learn more about how it works, visit the <a href="https://umap-learn.readthedocs.io/en/latest/" target="_blank">UMAP documentation</a>.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai-community/showcase/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>